doc/api.js

```diff
@@ -265,6 +265,7 @@
 
 /**
  * Async function to get the version of the nrfjprog DLL in use.
```

Comment:
I would add the word "alias" somewhere in here, add a `@linkcode{}` to `getLibraryVersion()` and delete the example.